litestar/dto/base_dto.py

```diff
@@ -139,12 +140,18 @@ def is_supported_model_type_field(cls, field_definition: FieldDefinition) -> boo
         )
 
     @classmethod
-    def create_for_field_definition(cls, field_definition: FieldDefinition, handler_id: str) -> None:
+    def create_for_field_definition(
+        cls,
+        field_definition: FieldDefinition,
+        handler_id: str,
+        backend_cls: type[DTOBackend] | None = None,
```

Comment:
Don't know if this is the best way to handle this, but I didn't want to add a feature flag to this method as it will be removed anyway. This was a bit more generic and felt like it could be useful in other ways.